lite/jupyterlite.py

```diff
@@ -0,0 +1,26 @@
+# ---
+# jupyter:
+#   kernelspec:
+#     display_name: Python (Pyodide)
+#     language: python
+#     name: python
+# ---
+
+# %% [markdown]
+# # `pyhf` in the browser
+
+# %% [markdown]
+# * To run the code, click on the first cell (gray box) and press <kbd>Shift</kbd>+<kbd>Enter</kbd> or click on the (Run) ▶ button to run each cell.
+# * Alternatively, from the `Run` menu select `Run All Cells`.
+#
+# You will need to copy and paste the following block into the top of each notebook.
```

Comment:
@henryiii can I ask you to review this? It seems that on the test deployment I have (https://pyhf.github.io/pyhf-tutorial/live/lab/index.html?path=jupyterlite.ipynb) I need to continually reinstall `pyhf` for it to be recognized in each new notebook. Is the reason you don't have to do this for https://github.com/henryiii/level-up-your-python because there are [so many built in packages for Pyodide now](https://pyodide.org/en/stable/usage/packages-in-pyodide.html)? 

So for example you already have `pandas` so in https://github.com/henryiii/level-up-your-python/blob/b4a9e7e7a7f0b1495b43f4f47c764e16a12e3981/notebooks/3.3%20Pandas.ipynb you can just 

```python
import pandas as pd
```

without reinstalling?

Reply:
Things in Pyodide's manifest don't need to be micropip/piplite installed. But third party packages do have to be, as far as I now. Adding wheels to `/pypi` will just make them available for pipelite / micropip installs, I think, and is mostly useful for custom binaries. Though you could check - it'd be `pip download --dest pypi pyhf`, placed in your noxfile. But I think that doesn't help what you want.

You'll notice that I have to install things like rich to use them.

Reply:
Thanks!